fluss-server/src/test/java/com/alibaba/fluss/server/replica/ReplicaManagerTest.java

```diff
@@ -562,6 +569,73 @@ void testPutKvWithOutOfBatchSequence() throws Exception {
                 DATA1_ROW_TYPE, records, Arrays.asList(expectedLogForData1, expectedLogForData2));
     }
 
+    @Test
+    void testPutKvWithDeleteNonExistsKey() throws Exception {
```

Comment:
This method is failed in my local with exception:

```
java.lang.IndexOutOfBoundsException
	at com.alibaba.fluss.memory.MemorySegment.getIntNativeEndian(MemorySegment.java:741)
	at com.alibaba.fluss.memory.MemorySegment.getInt(MemorySegment.java:714)
	at com.alibaba.fluss.record.DefaultLogRecordBatch.getRecordCount(DefaultLogRecordBatch.java:227)
	at com.alibaba.fluss.record.FileLogInputStream$FileChannelLogRecordBatch.getRecordCount(FileLogInputStream.java:161)
	at com.alibaba.fluss.server.replica.ReplicaManagerTest.testPutKvWithDeleteNonExistsKey(ReplicaManagerTest.java:620)
```

Reply:
I ran many rounds locally without encountering this issue. Can you check again in your local env with the new commit.

Reply:
After rebuilding the source code with the latest branch, the failure disappeared. I think local dirty classes cause this.